test-data/unit/check-abstract.test

```diff
@@ -410,6 +410,8 @@ class B(object, A): pass \
       # E: Cannot determine consistent method resolution order (MRO) for "B"
 
 [case testOverloadedAbstractMethod]
+from foo import *
```

Comment:
This chicanery is now endemic to the previously-existing tests, to make them all test overloads in stub files, so I don't have to introduce an implementation to each.